src/XeroClient.ts

```diff
@@ -185,8 +187,8 @@ export class XeroClient {
   }
 
   public async getClientCredentialsToken(): Promise<TokenSet> {
-    const { clientId, clientSecret, grantType } = this.config;
-    const result = await this.tokenRequest(clientId, clientSecret, { grant_type: grantType });
+    const { clientId, clientSecret, grantType, scopes } = this.config;
```

Comment:
It shouldn't be an issue but might be worth testing this change with the custom connections starter app to confirm `scopes` being `undefined` doesn't break anything

Reply:
👍  Tested in custom connections app w/ out scopes for a normal Custom Connections app

![Screen Shot 2021-08-05 at 1 49 51 PM](https://user-images.githubusercontent.com/2935387/128412414-1a1f6a38-b9de-414f-b09a-04c4736e7a1a.png)
![Screen Shot 2021-08-05 at 1 51 22 PM](https://user-images.githubusercontent.com/2935387/128412417-7dad6e25-3dcd-4e73-898f-b2f660558e66.png)